enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/IdentityVerificationRepository.java

```diff
@@ -39,17 +39,6 @@
 @Repository
 public interface IdentityVerificationRepository extends CrudRepository<IdentityVerificationEntity, String> {
 
-    @Modifying
```

Comment:
I believe that it is not needed anymore, when reset sets status to FAILED.

Reply:
Ok.